Updates to NPM packages

awead opened the pull request:
This updates all available packages to their latest minor patch versions. Major version changes were skipped.

mtribone commented:
![screen shot 2017-11-13 at 10 22 07 am](https://user-images.githubusercontent.com/4163828/32733170-931b8b44-c85c-11e7-9a0e-bd5bf4f4f894.png)

cam156 commented on app/forms/collection_form.rb:
```diff
@@ -17,10 +18,28 @@ def initialize(model, current_ability, request)
   end
 
+  def self.multiple?(field)
+    if [:title].include? field.to_sym
```
`if field.to_sym == :title` would be an easier way of saying this.

cam156 commented on db/schema.rb:
```diff
@@ -569,8 +569,8 @@
 
   add_foreign_key "curation_concerns_operations", "users"
   add_foreign_key "mailboxer_conversation_opt_outs", "mailboxer_conversations", column: "conversation_id", name: "mb_opt_outs_on_conversations_id"
-  add_foreign_key "mailboxer_notifications", "mailboxer_conversations", column: "conversation_id", name: "notifications_on_conversation_id_development"
```
This file should be excluded

mtribone replied:
oopsies. I forgot to check it out again after I migrated the database.

cam156 commented on spec/features/collection/edit_spec.rb:
```diff
@@ -76,13 +78,15 @@
       expect(page).to have_checked_field('Public')
       expect(page).to have_no_checked_field('Private')
       fill_in 'Title', with: updated_title
+      fill_in 'Subtitle', with: updated_subtitle
```
There are two check below after the Update button is clicked that I would expect to also see for the subtitle.  Make sure the original subtitle is not present.   Make sure the new subtitle is present.

awead committed:
Updates to NPM packages

This updates all available packages to their latest minor patch
versions. Major version changes were skipped.
